roles/pytest_project/defaults/main.yml

```diff
@@ -19,3 +19,5 @@ pytest_project_junit_output: junit.xml
 pytest_project_markers:
 pytest_project_command_args:
 pytest_project_ignore_errors: no
+pytest_local_env: false
```

Comment:
I think the name is not very descriptive. Would it make more sense to add a more explicit `pytest_run_tests: true` instead?